lib/Varien/Object.php

```diff
@@ -854,4 +854,21 @@ public function flagDirty($field, $flag = true)
         }
         return $this;
     }
+
+    /**
+     * @throw Exception
+     * return string
+     */
+    public function __toString()
+    {
+        if (Mage::getIsDeveloperMode()) {
+            if (PHP_SAPI != 'cli') {
+                return empty($this->getId()) ? '<div><b>warning_echo_object[' . get_class($this) . ']</b></div>' :
+                    '<div><b>warning_echo_object[' . get_class($this) . '#' . $this->getId() . ']</b></div>';
+            }
+            return empty($this->getId()) ? "\n" . 'warning_echo_object[' . get_class($this) . ']' . "\n" :
+                "\n" . 'warning_echo_object[' . get_class($this) . '#' . $this->getId() . ']' . "\n";
```

Comment:
```suggestion
            if (PHP_SAPI == 'cli') {
                return empty($this->getId()) ? "\n" . 'warning_echo_object[' . get_class($this) . ']' . "\n" :
                    "\n" . 'warning_echo_object[' . get_class($this) . '#' . $this->getId() . ']' . "\n";
            }
            return empty($this->getId()) ? '<div><b>warning_echo_object[' . get_class($this) . ']</b></div>' :
                '<div><b>warning_echo_object[' . get_class($this) . '#' . $this->getId() . ']</b></div>';
```

I think == ifs are more readable than != one :-)